spec/support/acceptance/prepare_host.rb

```diff
@@ -1,5 +1,10 @@
-def cleanup_zabbix
-  shell('rm -rf /etc/yum.repos.d/Zabbix*.repo; rm -rf /var/cache/yum/x86_64/*/Zabbix*; yum clean all --verbose') if fact('os.family') == 'RedHat'
+def prepare_host
+  if fact('os.family') == 'RedHat'
+    shell('rm -rf /etc/yum.repos.d/Zabbix*.repo; rm -rf /var/cache/yum/x86_64/*/Zabbix*; yum clean all --verbose')
+    if fact('os.release.major').to_i == 7
+      shell('sed -i "/nodocs/d" /etc/yum.conf')
+    end
```

Comment:
I really wanted the comment to survive.

Reply:
oh right, my bad. fixed it in https://github.com/voxpupuli/puppet-zabbix/pull/710